src/elements/select/Select.js

```diff
@@ -6,16 +6,26 @@ import { KeyCodes } from './../../utils/key-codes/KeyCodes';
 @Component({
     selector: 'novo-select',
     directives: [COMMON_DIRECTIVES],
-    inputs: ['options', 'placeholder', 'readonly'],
+    inputs: ['options', 'placeholder', 'readonly', 'footerConfig'],
     outputs: ['onSelect'],
     template: `
         <button (click)="toggleActive($event)" tabIndex="-1" type="button" [ngClass]="{empty: empty}">{{selected.label}}<i class="bhi-collapse"></i></button>
-        <ul class="novo-select-list" tabIndex="-1">
+        <ul class="novo-select-list" tabIndex="-1" [ngClass]="{footer: footerConfig}">
             <ng-content></ng-content>
             <li *ngFor="let option of options; let i = index" [ngClass]="{active: option.active}" (click)="onClickOption(option, i)" [attr.data-automation-value]="option.label">
               <span>{{option.label}}</span>
               <i *ngIf="option.active" class="bhi-check"></i>
             </li>
+            <div *ngIf="footerConfig" class="select-footer">
```

Comment:
When toggling the "ADD NEW" the footer renders off screen, making me scroll down to save. Possible to not have it scroll when showing the input?